### PR TITLE
feat(clean): Do not abort but continue

### PR DESCRIPTION
**Pull request by henrikstranneheim**
## Description

The seqencers always add a dir called `Myrun` when copying data to cg-nas. This makes the clean process abort. Now it will skip dirs not following the flow cell naming pattern.

### Added

-

### Changed

- Accept FlowCellError and continue cleaning

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b fix-clean-flow-cells -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/cli/clean.py [hunk 1]
```diff
@@ -274,11 +272,9 @@ def clean_flow_cells(context: CGConfig, dry_run: bool):
                 dry_run=dry_run,
             )
             clean_flow_cell_api.delete_flow_cell_directory()
-        except CleanFlowCellFailedError as error:
+        except (CleanFlowCellFailedError, FlowCellError) as error:
             LOG.error(repr(error))
-            exit_code = EXIT_FAIL
-    if exit_code:
-        click.Abort
+            continue
```
Comment (islean) [inline cg/cli/clean.py:277]: Will this work? Seems as if the every error in delete_flow_cell_directory is wrapped into a CleanFlowCellFailedError.

Comment (islean) [reply cg/cli/clean.py:277]: Or right! That error is raised in the instantiation?

**Review by islean — APPROVED** [review]
Should do the trick! We would perhaps want to sort out the erroneous paths instead but uncertain if we would gain anything from it.

**Review by diitaz93 — APPROVED** [review]
👍

**Comment by henrikstranneheim** [comment]
### cg clean flow-cells -d 👍 
```
[henrik.stranneheim@hasta:~] [S_base] 3m34s $ cg clean flow-cells -d
Flowcell Myrun does not follow the flow cell naming convention
FlowCellError('Flowcell Myrun does not follow the flow cell naming convention')
Trying to delete /srv/seqdata/seqdata/231215_A01901_0260_AHWGMCDSX7
```

**Comment by henrikstranneheim** [comment]
Deployed in production:

```
Log deploy... done.
cg, version 54.4.9
```

**Pull request merged**